client/app/worlds/worlds.controller.spec.js

```diff
@@ -132,4 +136,30 @@ describe('Controller: WorldsCtrl', function () {
     });
   });
 
+  describe('delete a world', function() {
+    var world;
+
+    beforeEach(function () {
+      world = { name: 'Warcraft' };
+    });
+
+    it('should delete a world on confirmation', function() {
+      spyOn(window, 'confirm').and.returnValue(true);
+
+      scope.deleteWorldOnUserConfirmation(world);
+      scope.$digest();
+
+      expect(scope.deleteWorldStatus).toBe(1);
```

Comment:
At some point, we should remove the magic number and replace it with a manifest constant.